### PR TITLE
[12.0][MIG] partner_prospect: Migration to 12.0

### PR DESCRIPTION
**Pull request by randall-vx**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I'm seeing that there's other migration in progress in #756. Why didn't you check?

File: sale_partner_prospect/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                    'child_ids.invoice_ids'))
+            partner.prospect = (
+                not sale_ids.filtered(
+                    lambda r: r.state not in
+                    ('draft', 'sent', 'cancel')) and
+                not invoice_ids.filtered(
+                    lambda r: r.type in ('out_invoice', 'out_refund')))
+
+    prospect = fields.Boolean(
+        string='Prospect', compute='_compute_prospect', default=False,
```
Comment (Nikul-Chaudhary) [inline sale_partner_prospect/models/res_partner.py:36]: @rcastro-tyc  no need to `default=False`

**Comment by randall-vx** [comment]
Hi @pedrobaeza, Yes, I reviewed it before making the PR, I decided to do it again only because of the date the PR was made and because I needed it for a database.

Regards,

**Comment by pedrobaeza** [comment]
But the PR is still active and the contributor is attending reviews, so no need to superseed the other. Please indicate in the other PR what are the remaining points and if the author doesn't answer, then we can superseed that one with other, but not this way. Closing this one.

**Pull request closed without merging**